src/views/TreeView.ts

```diff
@@ -641,6 +651,44 @@ export class InfluxDBTreeProvider implements vscode.TreeDataProvider<ITreeNode>
             const node = new Instance(instance, this.context, this)
             nodes.push(node)
         }
-        return nodes
+
+        return new Promise<ITreeNode[]>((resolve, reject) => {
+            // Try to load any CLI configurations and add them as available connections
+            childProcess.exec('influx config list --json', (error, stdout) => {
```

Comment:
I think this work should live in `Store.getInstances` - that's what the `Store` was designed to be: an interface around multiple data sources. This means that `getInstances` becomes async, but I think that's fine.